### ISSUE TITLE
Promote on https://pre-commit.com/hooks.html

### ISSUE THREAD

**Issue by hukkin** [executablebooks/web-compile#3]
We should make a PR here https://github.com/pre-commit/pre-commit.com to make scss-compile appear on https://pre-commit.com/hooks.html

**Comment by chrisjsewell** [comment]
Yep cheers, just "solidifying" it a bit, and also there's a bit of an open question about how it interacts with PR merging (https://github.com/executablebooks/sphinx-panels/pull/32#issuecomment-692791196)
